site/en_AU.all.json

```
@@ -480,5 +480,7 @@
   "user_error_wrong_pin_reset": { "other": "Click here to reset your PIN" },
   "user_error_too_many_pin_errors_help_page_link": { "other": "" },
   "shopping_error_age_restricted": { "other": "This content is age restricted." },
-  "shopping_error_close_button": { "other": "OK" }
+  "shopping_error_close_button": { "other": "OK" },
+
+  "accessibility_homepage": { "other": "Homepage" }
```

Just wondering if "accessibility" might be misread by clients as something they could/should be translating and so perhaps something like "wcag" might be better to scare them off?

```suggestion
  "wcag_homepage_h1": { "other": "Homepage" }
```

---

Could this not just use the title of the homepage?

---

I guess they could want them to be different? E.g. a page title of `Dope Festival 2021 | Home` while `h1` of `Welcome to the dopest festival eva!`